sale_order_line_date/README.rst

```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Sale order line date
+====================
+
+This module adds requested date to a sales order lines and propagate it to
+stock moves and procurements.
+When the requested date of the whole sale order is modified the requested date
+of the lines change to match.
+
+Usage
+=====
+
+Create a Quotation or a Sales Order and it fills the requested date in the sale
+order line
+
```

Comment:
Please could you explain a little bit more the usage? I tested but what must a expect?

![2017-06-13_1-57-49](https://user-images.githubusercontent.com/8736623/27060313-321ddc14-4fdc-11e7-8f43-4213f1230ff0.png)